solr/core/src/java/org/apache/solr/util/ThreadCpuTimer.java

```diff
@@ -74,7 +74,7 @@ public static boolean isSupported() {
   /**
    * Return the initial value of CPU time for this thread when this instance was first created.
    * NOTE: absolute value returned by this method has no meaning by itself, it should only be used
-   * when comparing elapsed time between this value and {@link #getCurrentCpuTimeNs()}.
+   * when comparing elapsed time between this value and {@link #getElapsedTimerCpuNs()}.
```

Comment:
This comment is not true - elapsed time is already relative to the start of the timer, whereas the startCpuTime is an absolute value. The javadoc should say that the comparison should be between two absolute values, ie. startCpuTime and threadTotalCpu.

Reply:
ah refactor tools...